functests/4_latency/latency.go

```diff
@@ -348,21 +353,30 @@ func extractLatencyValues(testName string, exp string, node *corev1.Node) string
 	return latencies
 }
 
-// checkMaximumEnvironLatencyValue should look for one of the following environment variables:
+// setMaximumLatencyValue should look for one of the following environment variables:
 // OSLAT_MAXIMUM_LATENCY: the expected maximum latency for all buckets in us
 // CYCLICTEST_MAXIMUM_LATENCY: the expected maximum latency for all buckets in us
 // HWLATDETECT_MAXIMUM_LATENCY: the expected maximum latency for all buckets in us
-func checkMaximumEnvironLatencyValue(testName string) error {
+// MAXIMUM_LATENCY: unified expected maximum latency for all tests
+func setMaximumLatencyValue(testName string) error {
 	if testName != strings.ToUpper(oslatTestName) &&
 		testName != strings.ToUpper(cyclictestTestName) &&
 		testName != strings.ToUpper(hwlatdetectTestName) {
 		return fmt.Errorf("testName variable has incorrect value %q", testName)
 	}
 
+	var err error
+	unifiedMaxLatencyEnv := os.Getenv("MAXIMUM_LATENCY")
+	if unifiedMaxLatencyEnv != "" {
+		maximumLatency, err = strconv.Atoi(unifiedMaxLatencyEnv)
+		return fmt.Errorf("err: %v the environment variable MAXIMUM_LATENCY has incorrect value %q", err, unifiedMaxLatencyEnv)
```

Comment:
not sure this is correct. If we have the var set we parse as int... and then we always fail?

Reply:
Copied it with the original mistake.
Thanks!